openshift/template.yaml

```diff
@@ -436,6 +438,11 @@ parameters:
     name: ENABLE_SOCIAL_AUTH
     value: "True"
 
+  - description: ENABLE_LOGIN_FORM
+    displayName: ENABLE_LOGIN_FORM
+    name: ENABLE_LOGIN_FORM
```

Comment:
@vinaybommana Great work, so this flag `ENABLE_LOGIN_FORM
    value: "False"` will enable or disable the the login form via the deployment template?